### PR TITLE
Fix the dockerfile after my esy changes

### PR DESCRIPTION
**Pull request by phated**
Was able to get some really nice cleanup in here too!

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,6 @@ LABEL vcs-url="https://github.com/grain-lang/grain"
 LABEL maintainer="philip@grain-lang.org"
 
 RUN apt-get update
-RUN apt-get install -y cmake=3.13.4-1
```
Comment (ospencer) [inline Dockerfile:9]: Probably don't need the line above anymore either huh?

**Review by ospencer — COMMENTED** [review]
This is sooooooo much better!

File: Dockerfile (hunk 1 above)
Comment (phated) [reply Dockerfile:9]: Probably not. I wasn't sure if that updated like gcc or something. Do you think I should remove it?

Comment (ospencer) [reply Dockerfile:9]: Yeah, let's remove it.

Comment (phated) [reply Dockerfile:9]: Amended and force-pushed.

**Review by ospencer — APPROVED** [review]
Woohoo!

**Pull request merged**